bin/main.rb

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
+require_relative '../lib/player.rb'
+require_relative '../lib/game.rb'
 
 def print_board(arr)
```

Comment:
You should move `print_board `to another file` Board.rb` where you store the board methods. You don't have to print but you can return a string and print it on main.rb

This way, you keep your code more organized.